test/resources/accounts.spec.js

```diff
@@ -0,0 +1,187 @@
+'use strict'
+
+const chai = require('chai')
+const { assert } = chai
+const rzpInstance = require('../razorpay')
+const mocker = require('../mocker')
+const equal = require('deep-equal')
+
+let mockRequest = {
+  "email": "gauriagain.kumar@example.org",
+  "phone": "9000090000",
+  "legal_business_name": "Acme Corp",
+  "business_type": "partnership",
+  "customer_facing_business_name": "Example",
+  "profile": {
+    "category": "healthcare",
+    "subcategory": "clinic",
+    "description": "Healthcare E-commerce platform",
+    "addresses": {
+      "operation": {
+        "street1": "507, Koramangala 6th block",
+        "street2": "Kormanagala",
+        "city": "Bengaluru",
+        "state": "Karnataka",
+        "postal_code": 560047,
+        "country": "IN"
+      },
+      "registered": {
+        "street1": "507, Koramangala 1st block",
+        "street2": "MG Road",
+        "city": "Bengaluru",
+        "state": "Karnataka",
+        "postal_code": 560034,
+        "country": "IN"
+      }
+    },
+    "business_model": "Online Clothing ( men, women, ethnic, modern ) fashion and lifestyle, accessories, t-shirt, shirt, track pant, shoes."
+  },
+  "legal_info": {
+    "pan": "AAACL1234C",
+    "gst": "18AABCU9603R1ZM"
+  },
+  "brand": {
+    "color": "FFFFFF"
+  },
+  "notes": {
+    "internal_ref_id": "123123"
+  },
+  "contact_name": "Gaurav Kumar",
+  "contact_info": {
+    "chargeback": {
+      "email": "cb@example.org"
+    },
+    "refund": {
+      "email": "cb@example.org"
+    },
+    "support": {
+      "email": "support@example.org",
+      "phone": "9999999998",
+      "policy_url": "https://www.google.com"
+    }
+  },
+  "apps": {
+    "websites": [
+      "https://www.example.org"
+    ],
+    "android": [
+      {
+        "url": "playstore.example.org",
+        "name": "Example"
+      }
+    ],
+    "ios": [
+      {
+        "url": "appstore.example.org",
+        "name": "Example"
+      }
+    ]
+  }
+}
+
+const BASE_URL = '/accounts',
+  API_VERSION = 'v2',
+  TEST_ACCOUNT_ID = 'acc_GRWKk7qQsLnDjX';
+
+describe('ACCOUNTS', () => {
+  it('Create an account', (done) => {
+
+    mocker.mock({
+      version: API_VERSION,
+      url: `${BASE_URL}`,
+      method: 'POST'
+    })
+
+    rzpInstance.accounts.create(mockRequest).then((response) => {
```

Comment:
@ankitdas13  What exactly are we testing here? If we are testing for response, then we should not Mock it. It defeats the purpose of testing.

Reply:
@1995navinkumar We are only testing API methods here by checking the URL and methods with mock url and method and we are following this in all API entity test cases.

Reply:
Got it. But IMO, since we are integration something, we would also need to ensure the integration actually works instead of just testing the implementation details. 
We can just write test cases for happy flows and check whether we get status 200 for all requests. We can take this up as a separate discussion as well.